src/ui/public/courier/fetch/request/doc.js

```diff
@@ -1,42 +1,45 @@
 import _ from 'lodash';
```

Comment:
I don't think lodash is used in this file anymore.

For all of these "lodash not here" things, we can just do another PR.